app/controllers/mixins/breadcrumbs_mixin.rb

```diff
@@ -26,7 +26,7 @@ def data_for_breadcrumbs
         end
       else
         # Append breadcrumb from title of the accordion (eg "Policies")
-        breadcrumbs.push(:title => accord_name, :key => accord_container, :action => "accordion_select") if features?
+        breadcrumbs.push(:title => accord_name, :key => accord_role, :action => "accordion_select") if features?
```

Comment:
I don't think it's safe to rely on the `accord_role` here, according to the `git log` of the `ApplicationController::Feature` the fallback for the container in `accord_hash` was `"#{accord_name}_accord"` where the `accord_name` was identical with the `name` if not set.

In some edge cases where the `accord_name` was set to something custom, but I removed these. So I think here you should go with `name` from the features hash and maybe an `"_accord"` suffix, but you should definitely not tangle the roles into this.

Reply:
:+1: